sutta_publisher/tests/unit/test_base_parser.py

```diff
@@ -82,8 +82,9 @@ def test_should_parse_json_to_html(base_parser):
     # Adding the hardcoded html_[head|tail] for styling of output .html file. Will be useful for demo only
     with open("output_test_html_from_test_base_parser.html", "w") as f:
         f.write(f"{html_head}{generated_html}{html_tail}")
-    with open("output_test_html_from_test_base_parser (without css).html", "w") as f:
-        f.write(generated_html)
+    for i, html in enumerate(generated_html):
+        with open(f"output_test_html_from_test_base_parser_(without css)_{i}.html", "w") as f:
+            f.write(html)
```

Comment:
this test should be rewritten from ground up or discarded altogether.  We shouldn't store test outputs in the repo. For now maybe just 
```python
assert generated_html is not None
```
?